src/lib.rs

```diff
@@ -1411,6 +1411,24 @@ where
     }
 }
 
+impl<L, R> fmt::Write for Either<L, R>
+where
+    L: fmt::Write,
+    R: fmt::Write,
+{
+    fn write_str(&mut self, s: &str) -> fmt::Result {
+        for_both!(*self, ref mut inner => inner.write_str(s))
```

Comment:
If you write this instead, can you do this without the `ref mut`?
```suggestion
        for_both!(self, inner => inner.write_str(s))
```

Reply:
Probably happened before match ergonomics were improved. I was gonna open a PR to simplify all those patterns, but have already opened 3 just today, so I'll wait with that. 😅